lib/plenty_client/request.rb

```diff
@@ -110,13 +110,13 @@ def base_url(path)
       # 2017-12-04 DO: there has to be a supervisor watching over the users limits
       #                BEFORE the request actually happens
       #                response_header is after the request and useless if you have multiple instances of the Client
-      def throttle_check_short_period(response_header)
+      def throttle_check_short_period(response_header, bandwidth = 1)
```

Comment:
$bandwidth, where is it used?

Reply:
My bad, I was discussing something with Michael and wrote this as an example... Removing...